contato.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+    <head>
+       <meta charset="UTF-8">
+       <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Contato</title>
+    </head>
+    <body>
+        <ul>
+            <li>
+                <a href="index.html">Home</a>
+            </li>
+            <li>
+                <a href="contato.html">Contato</a>
+            </li>
+        </ul>
+
+        <h1>Entre em contato</h1>
+
+        <form>
+            <label for="nome">Nome:</label> <br>
+            <input type="text" id="nome">
+            <br><br>
+            <label for="email">Email</label> <br>
+            <input type="email" id="email">
+            <br><br>
+            <label for="assunto">Assunto</label> <br>
+            <textarea name="assunto" id="assunto" cols="60" rows="10"></textarea>
+            <br><br>
+            <button>Enviar</button>
+        </form>
```

Comment:
trying the multiline comment
```suggestion
        <form>
            <label for="nome">Nome:</label> <br>
            <input type="text" id="nome">
            <br><br>
            <label for="email">Email</label> <br>
            <input type="email" id="email">
            <br><br>
            <label for="assunto">Assunto</label> <br>
            <textarea name="assunto" id="assunto" cols="60" rows="10"></textarea>
            <br><br>
            <button>Enviar</button>
        </form> suggestion(?)
```